tests/integ/test_inference_recommender.py

```diff
@@ -154,6 +154,92 @@ def advanced_right_sized_model(sagemaker_session, cpu_instance_type):
             )
 
 
+@pytest.fixture(scope="module")
+def default_right_sized_unregistered_model(sagemaker_session, cpu_instance_type):
```

Comment:
can we integ test the whole workflow from FrameworkModel() -> ... -> deploy()? we need to have confidence that after `right_sized()` that it can in turn `deploy()`.  We own this experiance